ios/FluentUI/Label/Label.swift

```diff
@@ -43,16 +43,20 @@ open class Label: UILabel, TokenizedControlInternal {
     }
 
     open override var textColor: UIColor! {
-        didSet {
-            labelTextColor = textColor
-            updateTextColor()
+        get {
+            return tokenSet[.textColor].uiColor
+        }
+        set {
+            tokenSet[.textColor] = .uiColor { newValue }
         }
     }
 
     open override var font: UIFont! {
-        didSet {
-            labelFont = font
-            updateFont()
+        get {
+            return tokenSet[.font].uiFont
```

Comment:
Are you going to try and address the actual issue we found with the `Label`'s font returning the incorrect value, or just revert the revert? Maybe changing this to `return super.font` would work, since we set `super.font` in `updateFont()`?

Reply:
If we're going to solve this issue, I think it's better suited for a separate PR. I'd argue that reverting the revert is fine for now.